Documentation/General/productVision.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\begin{document}
+
+\title{Product Vision\\
+	\begin{small}
+	The why, who and what
+	\end{small}
+}
+\author{MoodCat.me}
+\maketitle
+
+\section{Why should we build MoodCat?}
+Listening to music is currently a solo activity.
+Various competitors provide a system that can offer users popular songs or songs that are similar to songs the user already listened to.
+These systems don't offer users songs they haven't heard yet and that might fit the user's current need.
+Since these systems provide solely songs that fit only the user and are very limited in varience, a better solution can be achieved.
+
+Secondly there are some competitors that offer users a room to socialize while experiencing songs selected by other users in the room.
+While this might offer the user unknown songs, the system is user-driven and does not offer the listeners that want to listen on their own.
+
+\section{What are the selling points of MoodCat?}
+MoodCat has several selling points that makes it unique in the current music market.
+
+\begin{enumerate}
+\item MoodCat is self-maintaining, which makes it possible to handle audience sizes from one to one thousand.
+This makes it possible to use the system no matter how many other users are currently online.
+
+\item MoodCat offers users both to experience solely or communicate with fellow music listeners.
+
+\item MoodCat does not suffer a lot of a cold start.
+The system is able to offer relevant songs using the specified current mood of the user.
+
+\item MoodCat offers unknown/lesser known artists the possibility to promote their songs.
+The system will choose songs that are in the long tail of fewer-played songs in order to keep a high variety of songs.
+
+\item MoodCat can determine mood trends of a long-term user and can adjust the song-selection mechanism accordingly.
+Therefore long-term users are served more suitable songs which increases their overall satisfaction.
```

Comment:
While this might be the case for single-user rooms, we run into problems for rooms with multiple listeners. Also mood trends can be difficult to measure. Is the listener listening to music similar to his/hers mood? To what extend we actually cause a mood for our listener? I think this point might be a bit beyond our focus. I would suggest the following two options:
- When we see (through the votes) that a users mood/preference does not match the room at all, we suggest another room
- Adjust the rooms features/details accordingly to user input to better match the current mood of it's listeners